code/mathematical_algorithms/src/primality_tests/fermat_primality_test/fermat_primality_test.py

```diff
@@ -0,0 +1,13 @@
+import random
+
+def isPrime(n, k=10):
+    if n <= 1:
+        return False
+    if n <= 3:
+        return True
+
+    for i in range(k):
+        a = random.randint(2, n-1)
+        if pow(a, n-1, n) != 1:
+            return False
+    return True
```

Comment:
You should add a `main` function, called when `if __name__ == '__main__'`, that contains some tests.